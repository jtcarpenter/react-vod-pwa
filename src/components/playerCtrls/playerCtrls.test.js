import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {PlayerCtrls} from
    'components/playerCtrls/PlayerCtrls';

const mockPlayerCtrls = (
    <PlayerCtrls/>
)

describe('PlayerCtrls', () => {
    it('should render', () => {
        const wrapper = shallow(mockPlayerCtrls);
        expect(wrapper.exists()).toBe(true);
    });

    it('should display correctly', () => {
        const playerCtrls = renderer.create(mockPlayerCtrls);
        expect(playerCtrls.toJSON()).toMatchSnapshot();
    });
});
