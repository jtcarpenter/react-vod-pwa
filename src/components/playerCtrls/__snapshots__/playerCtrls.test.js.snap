// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerCtrls should display correctly 1`] = `
<div
  className="sc-bdVaJa kmXoBq"
/>
`;
